atm.js

```diff
@@ -1,21 +1,96 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
+$("input").eq(1).on("click", depositIntoChecking);
+$("input").eq(4).on("click", depositIntoSavings);
+$("input").eq(2).on("click", withdrawFromChecking);
+$("input").eq(5).on("click", withdrawFromSavings);
+$(".account").eq(0).addClass("zero");
+$(".account").eq(1).addClass("zero");
+$("input").eq(0).on("keypress", function(event) {
+  if(event.which == 13) {
+    event.preventDefault();
+    var checkingAccountUserInput = Number(parseFloat($("input").eq(0).val()));
+    var checkingTotalAfterWithdrawal = (parseFloat($("div.balance").eq(0).html().substr(1))) + checkingAccountUserInput;
+    $("div.balance").eq(0).html("$" + checkingTotalAfterWithdrawal.toFixed(2));
+    $("input").eq(0).val("");
+    $(".account").eq(0).removeClass("zero");
+  }
+});
+$("input").eq(3).on("keypress", function(event) {
+  if(event.which == 13) {
+    event.preventDefault();
+    var savingsAccountUserInput = Number(parseFloat($("input").eq(3).val()));
+    var savingsTotal = (parseFloat($("div.balance").eq(1).html().substr(1))) + savingsAccountUserInput;
+    $("div.balance").eq(1).html("$" + savingsTotal.toFixed(2));
+    $("input").eq(3).val("");
+    $(".account").eq(1).removeClass("zero");
+  }
+});
+function depositIntoChecking(event) {
+  event.preventDefault();
+  var checkingAccountUserInput = Number(parseFloat($("input").eq(0).val()));
+  var checkingTotalAfterWithdrawal = (parseFloat($("div.balance").eq(0).html().substr(1))) + checkingAccountUserInput;
+  $("div.balance").eq(0).html("$" + checkingTotalAfterWithdrawal.toFixed(2));
+  $("input").eq(0).val("");
+  $(".account").eq(0).removeClass("zero");
 }
-
-function checkingWithdrawal(amount){
-
+function depositIntoSavings(event) {
+  event.preventDefault();
+  var savingsAccountUserInput = Number(parseFloat($("input").eq(3).val()));
+  var savingsTotal = (parseFloat($("div.balance").eq(1).html().substr(1))) + savingsAccountUserInput;
+  $("div.balance").eq(1).html("$" + savingsTotal.toFixed(2));
+  $("input").eq(3).val("");
+  $(".account").eq(1).removeClass("zero");
 }
-
-function savingsDeposit(amount){
-
+function withdrawFromChecking(event) {
+  event.preventDefault();
+  var checkingAccountUserWithdrawal = Number(parseFloat($("input").eq(0).val()));
+  var checkingTotalAfterWithdrawal = (parseFloat($("div.balance").eq(0).html().substr(1))) - checkingAccountUserWithdrawal;
```

Comment:
I'd also extract out the complex part here into another variable, something like:
```js
var currentBalance = parseFloat($("div.balance").eq(0).html().substr(1));
var checkingTotalAfterWithdrawal = currentBalance - checkingAccountUserWithdrawal;
```